pynvjitlink/tests/test_pynvjitlink.py

```diff
@@ -49,30 +49,26 @@ def undefined_extern_cubin():
     return read_test_file('undefined_extern.cubin')
 
 
-@pytest.mark.skip
```

Comment:
I think this was fixed in CTK 12.2, but we're not installing environments with that yet, so I am not sure if we can skip these?

Reply:
We require cuda 12.3 to build the project now https://github.com/rapidsai/pynvjitlink/blob/main/CMakeLists.txt#L14

Reply:
Ah, OK. Is that also the version that the end user gets?

Reply:
Yes - we build in the two pieces of the CTK that are used statically so the user should end up with the 12.3 ctk binaries on their system regardless of what they have installed.